effects/src/main/scala/com/github/gvolpe/fs2redis/interpreter/Fs2Redis.scala

```diff
@@ -779,6 +780,11 @@ private[fs2redis] class BaseFs2Redis[F[_], K, V](
       conn.async.flatMap(c => F.delay(c.zscore(key, value)))
     }.map(x => Option(Double.box(x)))
 
+  /******************************* Connection API **********************************/
+  override val ping: F[String] =
+    JRFuture {
+      conn.async.flatMap(c => F.delay(c.ping()))
```

Comment:
I wanted to add a variant that takes an optional String parameter (the redis docs claim it's possible to send a custom ping message), but lettuce doesn't allow that.